charts/spire/README.md

```diff
@@ -32,6 +32,16 @@ For production installs, please see [the production example](https://github.com/
 
 ## Upgrade notes
 
+### 0.15.X
+
+The spire-crds chart has been updated. Please ensure you have upgraded spire-crds before upgrading the spire chart.
+
+The chart now includes support for multiple parallel installs of the spire-controller-manager. To facilitate this change, all custom resources now take
+a className property. Out of the box the chart now uses and requires it to be set. If you have not loaded any yourself, the upgrade process will be transparent.
+
+If you have loaded your own custom resources, set spire-server.controllerManager.watchClassless=true until you can update your custom resources to have the
+className for the instance specified.
```

Comment:
I think we need to explain this new feature in the first paragraph so people can follow allong. I have difficulty following this entire section as I miss the context of the entire feature.

Reply:
An attempt at that was:
> The chart now includes support for multiple parallel installs of the spire-controller-manager.

Should we put more description then that int he upgrade notes, or a separate section? Was trying to keep the upgrade notes concise for those not interested in the feature.